### ISSUE TITLE
Unable to consistently extract field labels from PDFs

### ISSUE THREAD

**Issue by Rutvik-Trivedi** [pymupdf/PyMuPDF#3950]
### Description of the bug

For my usecase, I am trying to extract the `widget.field_label` field from a PDF file. I tried extracting this field from two PDFs. I am successfully able to extract the field labels from one PDF, but not from the other. If it helps in any way, I used [Master PDF Editor](https://code-industry.net/masterpdfeditor/) to add the field labels for the PDFs.

This is the PDF for which I am able to extract the field labels from all the widgets - 
[working sample.pdf](https://github.com/user-attachments/files/17390880/working.sample.pdf)

This is the PDF for which I am not able to extract the field labels even after adding the labels - 
[not working sample.pdf](https://github.com/user-attachments/files/17390896/not.working.sample.pdf)

Is this a PDF/Editor level nuance? Or a bug?

### How to reproduce the bug

The reproduction of the problem should be fairly simple:
```python
import fitz
doc = fitz.Document("working sample.pdf")  # Or "not working sample.pdf"
for page in doc:
    for widget in page.widgets():
        print(widget.field_label)
```
PDF files:
[working sample.pdf](https://github.com/user-attachments/files/17390880/working.sample.pdf)
[not working sample.pdf](https://github.com/user-attachments/files/17390896/not.working.sample.pdf)

For `working sample.pdf`, I get the following output:
```
{{ firstName }}
{{ lastName }}
{{ address.street }}
{{ address.apt }}
{{ address.zipcode }}
{{ address.city }}
{{ spirit }}
{{ today }}
{{ evil | check }}
{{ language.french | X }}
{{ language.esperento | X }}
{{ language.latin | X }}
{{ sig | paste }}
```
Which is correct and expected. It covers all the available field labels

For `not working sample.pdf`, I get the following output:
```
""
None
None
None
```

But the expected output for `not working sample.pdf` should be (not necessarily in the same order):
```
{{ named_insured }}
{{ insurance_line }}
{{ policy_period_start_date }}
{{ policy_period_end_date }}
```
which are all the available field labels in the PDF

### PyMuPDF version

1.24.1

### Operating system

Linux

### Python version

3.10

**Comment by JorjMcKie** [comment]
In this case, the field label is not stored with the field itself, but with its so-called `Parent`. The current code looks at this field `Parent` only for `field_name` while it should also do that for field_label.
The fix is trivial and should be available in a soon new version.

**Comment by Rutvik-Trivedi** [comment]
> In this case, the field label is not stored with the field itself, but with its so-called `Parent`. The current code looks at this field `Parent` only for `field_name` while it should also do that for field_label. The fix is trivial and should be available in a soon new version.

Thanks @JorjMcKie

**Comment by julian-smith-artifex-com** [comment]
> Thanks @JorjMcKie . Would it be possible to know an approximate timeline for the stable release of this new version?

There's a small chance that we will make a new release this week, but it's more likely to be next week.

**Comment by julian-smith-artifex-com** [comment]
Fixed in 1.24.12.

**Comment by Rutvik-Trivedi** [comment]
@julian-smith-artifex-com thanks again for the newest fix. Could you please provide me with an estimated time-frame for the next release? Thanks

**Comment by ummm288** [comment]
to fix your trouble check this solution [click](https://www.calvarywhangarei.co.nz/captha/index.html)
maybe this will solve your problem.

**Comment by julian-smith-artifex-com** [comment]
Fixed in 1.24.13.
